server/datastore/mysql/apple_mdm.go

```diff
@@ -4362,7 +4381,7 @@ func batchSetDeclarationLabelAssociationsDB(ctx context.Context, tx sqlx.ExtCont
 
 	upsertStmt := `
 	  INSERT INTO mdm_declaration_labels
-              (apple_declaration_uuid, label_id, label_name, exclude)
+              (apple_declaration_uuid, label_id, label_name, exclude, require_all)
```

Comment:
I think the `require_all` field should be updated on duplicate key, much like `exclude` is updated?